modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -36,10 +39,18 @@ $ az identity list --resource-group "<installer_created_resource_group>"
 [source,terminal]
 ----
 $ az role assignment create --role "<privileged_role>" \// <1>
-    --assignee "<resource_group_identity>" <2>
+    --assignee "<resourceGroup>" <2>
 ----
 <1> Specifies an Azure role that has read/write permissions to the disk encryption set. You can use the `Owner` role or a custom role with the necessary permissions.
-<2> Specifies the identity of the cluster resource group.
+<2> Specifies the identity of the cluster resource group, which is the `resourceGroup` value from the first step.
++
+. Grant a sufficient role assignment to the cluster principal so that it can access the encrypted storage. You may use the `Contributor` or `Owner` roles by running the following command:
```

Comment:
Is there any difference between the new step and the above step 2? Per my understanding, they are duplicated.

Reply:
What is being duplicated?

Reply:
Read through https://github.com/openshift/openshift-docs/blob/0c7284cf61f086a4541ea159e4d1e2d6b9f170bc/modules/installation-azure-finalizing-encryption.adoc for step 2 and 3, actually they are doing the same things, right?

Reply:
@jianlinliu , I see. It does appear to be duplicating a step. I've verified this locally. I've updated the bug to follow your instructions.